pom.xml

```diff
@@ -755,13 +789,35 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>jdk8</id>
+            <activation>
+                <jdk>8</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.source>1.8</maven.compiler.source>
+                <maven.compiler.target>1.8</maven.compiler.target>
+            </properties>
+        </profile>
         <profile>
             <id>jdk9</id>
             <activation>
                 <jdk>9</jdk>
             </activation>
             <properties>
                 <modules.argline>--add-modules java.xml.bind</modules.argline>
+                <maven.compiler.source>9</maven.compiler.source>
+                <maven.compiler.target>9</maven.compiler.target>
+            </properties>
+        </profile>
+        <profile>
```

Comment:
Do we actually need all these profiles? How about the suggestion above: keep JDK 8 compatibility at runtime, but require JDK 11 or later at build time (can be required via the Maven enforcer plug-in)?

Reply:
Nope. I think I can just update the jdk9 profile to be 11 and then the Travis build can use that Profile for builds. Then everyone else can continue with java 8 for now. 

- [ ] Clean up profiles.